test/test_translator.py

```diff
@@ -4,9 +4,6 @@
 import xmltodict
 import json
 from jsonschema import validate
-from datetime import datetime
-import subprocess
-
 
 
 
```

Comment:
We're still missing edge case testing in this file overall. We need to add in cases to test where no data is passed to the function, or bad data is.

Also, unit tests should be named so we know what they are testing and not just the function being tested. For example the polyline test below we could name `test_parse_polyline_valid_points` then if we test with no data, `test_parse_polyline_no_data`.

Reply:
Added some more edge cases and pushed the updated code to dev